config/_default/menus/main.en.yaml

```
@@ -1619,6 +1619,11 @@ menu:
       parent: dashboards
       weight: 9
       identifier: dashboards_overlays
+    - name: Annotations
+      url: dashboards/annotations/
+      parent: dashboards
```

If you want this to be a child of Overlays, you just need to change the parent identifier. If you want this to be the same level as Overlays, then the weight needs to be 10 (and then the following menu items weights need to be adjusted). Hope that helps, let me know if you have any questions!

```suggestion
      parent: dashboards_overlays
```